tests/fixture/tmpnet/README.md

```diff
@@ -57,32 +57,32 @@ repositories.
-| flags/common.go             |             | Defines type definitions common across other files                     |
-| flags/process_runtime.go    |             | Defines flags configuring the process node runtime                     |
-| flags/runtime.go            |             | Defines flags configuring node runtime                                 |
-| flags/start_network.go      |             | Defines flags configuring network start                                |
-| tmpnetctl/                  |             | Directory containing main entrypoint for tmpnetctl command             |
-| check_monitoring.go         |             | Enables checking if logs and metrics were collected                    |
-| defaults.go                 |             | Defines common default configuration                                   |
-| detached_process_default.go |             | Configures detached processes for darwin and linux                     |
-| detached_process_windows.go |             | No-op detached process configuration for windows                       |
-| flags.go                    | FlagsMap    | Simplifies configuration of avalanchego flags                          |
-| genesis.go                  |             | Creates test genesis                                                   |
-| kube.go                     |             | Library for Kubernetes interaction                                     |
-| local_network.go            |             | Defines configuration for the default local network                    |
-| monitor_processes.go        |             | Enables collection of logs and metrics from local processes            |
-| network.go                  | Network     | Orchestrates and configures temporary networks                         |
-| network_config.go           | Network     | Reads and writes network configuration                                 |
-| network_test.go             |             | Simple test round-tripping Network serialization                       |
-| node.go                     | Node        | Orchestrates and configures nodes                                      |
-| node_config.go              | Node        | Reads and writes node configuration                                    |
-| node_process.go             | NodeProcess | Orchestrates node processes                                            |
-| start_kind_cluster.go       |             | Starts a local kind cluster                                            |
-| subnet.go                   | Subnet      | Orchestrates subnets                                                   |
-| utils.go                    |             | Defines shared utility functions                                       |
+| Filename                    | Types          | Purpose                                                                |
```

Comment:
(out of scope) another thought: don't we want to just link to the [godoc page](https://pkg.go.dev/github.com/ava-labs/avalanchego/tests/fixture/tmpnet) instead of maintaining this table? 🤔